src/Features/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -179,5 +179,13 @@ public static ImmutableArray<AbstractFormattingRule> GetFormattingRules(this Doc
 
             return rules.AddRange(Formatter.GetDefaultFormattingRules(document));
         }
+
+        public static bool IsRazorDocument(this Document document)
```

Comment:
Method was moved

Reply:
You moved it here for ExternalAccess.Razor right? If you all are "ok" with this living here then I don't have an issue but I also wouldn't be opposed to duplicating this in the ExternalAccess.Razor layer

Reply:
Created a new ExternalAccess class for Razor extensions - https://github.com/dotnet/roslyn/blob/b9eb126fc9c7539f6e7cd3ee31dd1da318ac2f1c/src/Tools/ExternalAccess/Razor/Extensions.cs

Hope this is a little cleaner!